algebras/algebra/src/main/scala/endpoints/algebra/Requests.scala

```diff
@@ -125,6 +125,41 @@ trait Requests extends Urls with Methods with SemigroupalSyntax {
     */
   def textRequest: RequestEntity[String]
 
+  /**
+    * Alternative between two possible request entities, differentiated by the
+    * `Content-Type` header
+    *
+    * @note If [[A]] and [[B]] are both JSON-encoded and use disjoint schemas, use
+    *       [[endpoints.algebra.JsonSchemas.TaggedOps#orElse]] at the schema level instead
+    *
+    * Server interpreters accept either of the request entities
+    * Client interpreters provide one of the two request entities
+    * Documentation interpreters list all possible content types and their entities
+    */
+  def choiceRequestEntity[A, B](
+      requestEntityA: RequestEntity[A],
+      requestEntityB: RequestEntity[B]
+  ): RequestEntity[Either[A, B]]
+
+  implicit class RequestEntitySyntax[A](requestEntity: RequestEntity[A]) {
```

Comment:
Could you also add that the type `RequestEntity` has implicit methods provided by the `RequestEntitySyntax` class, on line 98?